agent/agent-tooling/src/main/java/io/opentelemetry/sdk/metrics/internal/view/PreAggregatedStandardMetrics.java

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package io.opentelemetry.sdk.metrics.internal.view;
+
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.sdk.metrics.InstrumentSelector;
+import io.opentelemetry.sdk.metrics.SdkMeterProviderBuilder;
+import io.opentelemetry.sdk.metrics.View;
+import io.opentelemetry.sdk.metrics.ViewBuilder;
+import io.opentelemetry.sdk.metrics.ViewBuilderAccessor;
+import io.opentelemetry.semconv.trace.attributes.SemanticAttributes;
+import java.util.HashSet;
+import java.util.Set;
+
+@SuppressWarnings("rawtypes")
+public class PreAggregatedStandardMetrics {
+
+  private static final Set<AttributeKey> HTTP_CLIENT_VIEW = buildHttpClientView();
+  private static final Set<AttributeKey> HTTP_SERVER_VIEW = buildHttpServerView();
+  private static final Set<AttributeKey> RPC_CLIENT_VIEW = buildRpcClientView();
+  private static final Set<AttributeKey> RPC_SERVER_VIEW = buildRpcServerView();
+
+  public static void registerViews(SdkMeterProviderBuilder builder) {
```

Comment:
Could be part of a `ViewRegistry` object: separation of view creation from view registration